Src/ILGPU.Algorithms/SequenceExtensions.cs

```diff
@@ -214,11 +214,11 @@ public static class SequenceExtensions
         /// <param name="accelerator">The accelerator.</param>
         /// <returns>The loaded sequencer.</returns>
         private static Action<
-            AcceleratorStream,
-            LongIndex1D,
-            SequenceImplementation<T, TStride, TSequencer>>
+                AcceleratorStream,
```

Comment:
QUESTION: Are we able to align to a single indentation relative to the previous line? This looks like it is aligning to the `static` keyword.

Reply:
could you share with me some keyword about this implementation?
to be transparent, I'm not from Dotnet world :D 
but what i do, I'm testing locally your suggestion with finding possible match within rules list
when i say rules, it's XML format for `jb cleanup` code and the one I'm using is this: https://razer-rbi.github.io/resharper-configurator/#

Reply:
Using the v1 rules, the code:
```CSharp
private static Action<
    AcceleratorStream,
    LongIndex1D,
    SequenceImplementation<T, TStride, TSequencer>>
    CreateRawSequencer<T, TStride, TSequencer>(
```
was reformatted to:
```CSharp
private static Action<
        AcceleratorStream,
        LongIndex1D,
        SequenceImplementation<T, TStride, TSequencer>>
        CreateRawSequencer<T, TStride, TSequencer>(
        this Accelerator accelerator)
```


It looked like the formatter was aligning `AcceleratorStream` to `static` on the first line.

I'm not sure what the rule would be called - I was surprised that it indented the code in the first place.

Reply:
at the moment, i couldn't find anything related to this
could be that `.editorconfig` has conflicts with this and during push it change files

Reply:
Let's leave it for now, and will see what the impact is like across all the files. Thanks.